ReactiveXComponent/Connection/IXCSubscriber.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using ReactiveXComponent.RabbitMq;
+
+namespace ReactiveXComponent.Connection
+{
+    public interface IXCSubscriber : IDisposable
+    {
+        void Subscribe(string component, string stateMachine, Action<MessageEventArgs> stateMachineListener);
+        void Unsubscribe(string component, string stateMachine, Action<MessageEventArgs> stateMachineListener);
+        IObservable<MessageEventArgs> GetStateMachineUpdates();
```

Comment:
You should expose a property rather than a method for the IObservable. Something like _StateMachineUpdatesStream_